### DISCUSSION TITLE
此楼可以讨论目前大家使用 hy2 中遇到的各种问题，提供样本

### DISCUSSION THREAD

**Discussion by stqfdyr** [HyNetworks/hysteria#1160] [General]
我先来，我目前使用正常，没有出现被阻断的现象，测速也正常，用的是 brutal ，没有用 bbr，可能和地域有关？

**Comment by h1-gwakpe3** [comment]
今天我搭建的日本，美国两个节点同时崩了

**Comment by xtccc** [comment]
我单443被封了 ，换了大范围端口跳跃 没啥问题

**Reply by xtccc** [reply]
端口跳跃好像也不行了，总是报 no recent network activity

**Reply by blackboxs777** [reply]
是的，一开始用的好好的，第二天就崩了，应该是被墙识别到了，需要更好的策略

**Comment by FFFKLD** [comment]
4431端口被封，伪装网址bing.com，使用iptable进行20000-50000的端口转发，无效

**Comment by stqfdyr** [comment]
昨天我也中招了（2024-7-18），不过今天的情况突然有了很大改善（2024-7-19）

**Comment by KEVIN-G386** [comment]
确认服务器ip被标记，回去的udp包直接被丢，tcp正常，海外服务器udp也正常，测试的几个端口都是如此

**Reply by blackboxs777** [reply]
对的，我同时搭建的reality 和hy2 ，hy2崩了，reality 还存货

**Comment by h1-gwakpe3** [comment]
我最早有两台部署hy2的服务器（东京的亚马逊云和凤凰城的甲骨文云）
大概在搭建好五天左右被封
昨天我重新搭建开了一个服务器今天中午被封

**Reply by Xuyuanjin** [reply]
校园网2天就屏蔽了VPS的ip，手机流量还可以使用。

**Reply by iddsjo** [reply]
> 校园网2天就屏蔽了VPS的ip，手机流量还可以使用。

TCP协议能用吗？有可能是被运营商屏蔽了

**Comment by lyc8503** [comment]
服务器对应端口 UDP 回包被 100% 丢弃，换端口之后可以缓解一段时间，但之后还是会被封禁。

所用服务器：Azure & Sharon.io，都观察到这个现象，配置如下，今日封禁症状似乎仍然没有显著缓解。

```
listen: :1234
speedTest: true

acme:
  domains:
    - xxx.xxx.cf
  email: xxxx@gmail.com

auth:
  type: password
  password: xxxxxx

masquerade: 
  type: proxy
  proxy:
    url: https://news.ycombinator.com/ 
    rewriteHost: true
```

**Reply by Hcc422** [reply]
我一个用端口跳跃暂时正常，一个换什么端口都不行，考虑是不是要换vless了。

**Reply by powerjiayun** [reply]
> 我一个用端口跳跃暂时正常，一个换什么端口都不行，考虑是不是要换vless了。

我也遇到，端口跳跃有个节点不管用，换ipv6方式继续使用hy2解决了

**Reply by toyo2333** [reply]
> 我一个用端口跳跃暂时正常，一个换什么端口都不行，考虑是不是要换vless了。

端口跳跃是不是就不加密了？

**Reply by gaojunke** [reply]
> > 我一个用端口跳跃暂时正常，一个换什么端口都不行，考虑是不是要换vless了。
> 
> 我也遇到，端口跳跃有个节点不管用，换ipv6方式继续使用hy2解决了

请问如何换成IPv6的地址？

**Comment by cancundeyingzi** [comment]
多台服务器,,一连上所有udp全部封掉。(比方说abcd4台。只要连了a,用一会儿。Abcd第四台全部超时。不知道过多久之后会恢复一下，然后一使用又超时。)四台服务器地区运营商都不同。客户机是移动。

**Reply by cancundeyingzi** [reply]
忘记说了端口跳跃全开。

**Reply by Sakamakiiizayoi** [reply]
俺也一样

**Reply by xingpingcn** [reply]
和客户端的ISP关系挺大，我hy2和vless（分别部署在两台不同ip的机器）有概率会同时阻断，马上换了isp就没有问题

**Reply by Sakamakiiizayoi** [reply]
自从换了30005端口已经稳定用几个月了 不知道是不是换端口的原因，最近出现过几次端口封禁不过很快就恢复了

**Reply by Sakamakiiizayoi** [reply]
当我没说 又开始抽风了

**Reply by yjd** [reply]
最近一个月也出现这情况。a用hy然后访问不了。再连其他2台也同样这情况。这时候换tcp的代理就没问题。或者停一段时间可以恢复正常访问hy，还有就是最近用的这个a是刚买bwg的gia 不知道是不是ip太脏被标记导致这情况。

**Reply by Sakamakiiizayoi** [reply]
用hy2还得是ipv6  最近买了ak家的日本iij 这是我第一次用到优化线路的ipv6 用了两个月了 稳如老狗没有任何断流限速 晚上油管都能跑个四五十万（用的bbr 没有用Brutal）这还是在100m带宽下跑的 感觉翻墙已经到头了 ipv4的话高峰期表现就差远了 总之就是ipv6感觉墙都不怎么管 我甚至直接用过ss一段时间 没有被封 只是翻墙表现没有hy2好就不用了

**Reply by yjd** [reply]
> 最近一个月也出现这情况。a用hy然后访问不了。再连其他2台也同样这情况。这时候换tcp的代理就没问题。或者停一段时间可以恢复正常访问hy，还有就是最近用的这个a是刚买bwg的gia 不知道是不是ip太脏被标记导致这情况。

经过我一天的使用，确定是买的bwg-gia ip被标记。长时间使用其他台用hy不会被阻断。bwg就会用一段时间hy被阻断，然后停一段时间又能用。tcp不会任何时间都不会。

**Comment by devnakx** [comment]
ipv4发送出去的udp包被阻断了，但是能收包，换端口也不行，不定时这样阻断
刚刚试了下Ipv6，也这样，不过端口跳跃可以暂时解决，但过一会又会被封
用的搬瓦工vps，dc9机房，ipv4 cn2 gia线路
![1](https://github.com/user-attachments/assets/24bef076-f1b7-406b-b182-ed29e3a24b9e)

**Reply by rebelliouswhiz** [reply]
你是怎么用 IPv6 测试 IPv4-only 服务器的？👀

**Reply by devnakx** [reply]
？我啥时候说我这vps是ipv4-only了？

**Reply by devnakx** [reply]
bwg几周前就支持给vps分配ipv6地址了，但是没有走cn2线路，我还发工单问了，回复原话是"We are looking into it and hope it will come in the near future. There is no ETA, but we are looking into it."

**Reply by rebelliouswhiz** [reply]
Alright, didn't know. Good for you.

**Reply by yjd** [reply]
现在一些地区开始对xyz域名做限制。如果你用这个，换别的就没问题。我的问题最后测试结果是这个问题

**Reply by devnakx** [reply]
和域名无关，我iperf3域名和IP地址测都一样的结果

**Reply by devnakx** [reply]
今天刚又买一个bwg-dc1的vps，刚配置好连接上去被秒阻断，我服了

**Reply by roymecat** [reply]
我用的阿里云SG和JP也搭建了hy2服务，很奇怪的是使用电信宽带连接基本上间歇性阻断，但5G电信卡连接似乎没有问题

严重怀疑GFW已经在试点hy2协议精准阻断了

**Reply by roymecat** [reply]
我开的端口是443，尝试大范围端口跳跃仍旧精准阻断

**Reply by Rangtian** [reply]
GIA没必要用HY2吧。。

**Reply by devnakx** [reply]
> GIA没必要用HY2吧。。

为什么没必要？

**Reply by toyo2333** [reply]
> > GIA没必要用HY2吧。。
> 
> 为什么没必要？

因为价格是你10分之一的烂线路，如果用HY2，最后速度和质量和你的GIA是一样的。反而，你好的GIA线路，因为用了HY2，还会遇到各种UDP QOS相关的症状。咋说呢。。。

属于没事找事。。。

**Reply by devnakx** [reply]
> > > GIA没必要用HY2吧。。
> > 
> > 
> > 为什么没必要？
> 
> 因为价格是你10分之一的烂线路，如果用HY2，最后速度和质量和你的GIA是一样的。反而，你好的GIA线路，因为用了HY2，还会遇到各种UDP QOS相关的症状。咋说呢。。。
> 
> 属于没事找事。。。

我这两个 vps 节点在之前一直运行 HY2 稳稳的，阻断是最近大概半年内的事，后来因为老是被间歇性阻断我就在 vps 上陆续尝试部署了trojan/vmess/vless（现在正在运行），速度都跑不满，多线程塞满了也只能跑到 50Mbps 左右，不如 HY2 一根，所以这个`没必要`因何说起？

**Reply by Rangtian** [reply]
> > > > GIA没必要用HY2吧。。
> > > 
> > > 
> > > 为什么没必要？
> > 
> > 
> > 因为价格是你10分之一的烂线路，如果用HY2，最后速度和质量和你的GIA是一样的。反而，你好的GIA线路，因为用了HY2，还会遇到各种UDP QOS相关的症状。咋说呢。。。
> > 属于没事找事。。。
> 
> 我这两个 vps 节点在之前一直运行 HY2 稳稳的，阻断是最近大概半年内的事，后来因为老是被间歇性阻断我就在 vps 上陆续尝试部署了trojan/vmess/vless（现在正在运行），速度都跑不满，多线程塞满了也只能跑到 50Mbps 左右，不如 HY2 一根，所以这个`没必要`因何说起？

搬瓦工GIA不该只有50Mbps。你不是电信宽带？

**Reply by devnakx** [reply]
就是电信啊，我自己就在用的我还不知道？这些基于 tcp 的代理协议全都 ± 50Mbps 雷打不动，只有 hy2 才能直接拉满宽带上限

**Reply by roymecat** [reply]
同电信用户，一到夜间什么线路都炸
UDP又老是被针对.....
已经准备换运营商了

**Reply by lovesans** [reply]
> 同电信用户，一到夜间什么线路都炸 UDP又老是被针对..... 已经准备换运营商了

不用换了,移动联通玩的更狠

**Reply by toyo2333** [reply]
> > > > GIA没必要用HY2吧。。
> 
> > > 
> 
> > > 
> 
> > > 为什么没必要？
> 
> > 
> 
> > 因为价格是你10分之一的烂线路，如果用HY2，最后速度和质量和你的GIA是一样的。反而，你好的GIA线路，因为用了HY2，还会遇到各种UDP QOS相关的症状。咋说呢。。。
> 
> > 
> 
> > 属于没事找事。。。
> 
> 
> 
> 我这两个 vps 节点在之前一直运行 HY2 稳稳的，阻断是最近大概半年内的事，后来因为老是被间歇性阻断我就在 vps 上陆续尝试部署了trojan/vmess/vless（现在正在运行），速度都跑不满，多线程塞满了也只能跑到 50Mbps 左右，不如 HY2 一根，所以这个`没必要`因何说起？

电信用户，用GIA搬瓦工一定可以跑满（别说搬瓦工这种牛逼的线路了，我用普通的9929来对付都可以跑满。跑不满，就是你的方案和设置有问题。

**Reply by devnakx** [reply]
> > > > > GIA没必要用HY2吧。。
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > 为什么没必要？
> > 
> > 
> > > 
> > 
> > 
> > > 因为价格是你10分之一的烂线路，如果用HY2，最后速度和质量和你的GIA是一样的。反而，你好的GIA线路，因为用了HY2，还会遇到各种UDP QOS相关的症状。咋说呢。。。
> > 
> > 
> > > 
> > 
> > 
> > > 属于没事找事。。。
> > 
> > 
> > 我这两个 vps 节点在之前一直运行 HY2 稳稳的，阻断是最近大概半年内的事，后来因为老是被间歇性阻断我就在 vps 上陆续尝试部署了trojan/vmess/vless（现在正在运行），速度都跑不满，多线程塞满了也只能跑到 50Mbps 左右，不如 HY2 一根，所以这个`没必要`因何说起？
> 
> 电信用户，用GIA搬瓦工一定可以跑满（别说搬瓦工这种牛逼的线路了，我用普通的9929来对付都可以跑满。跑不满，就是你的方案和设置有问题。

典

**Reply by doughnutscranberry** [reply]
https://github.com/net4people/bbs/issues/468
https://github.com/net4people/bbs/issues/458

**Reply by toyo2333** [reply]
> [net4people/bbs#468](https://github.com/net4people/bbs/issues/468) [net4people/bbs#458](https://github.com/net4people/bbs/issues/458)

没办法，上面都说的很直白了。HY2只适合作为备用方案用于象Racknerd或者新加坡甲骨文这种烂线路临时救急下。当然了，他可能是炫富哈~很多人还是买不起CN2 GIA的。

无所谓了，QUIC类协议，进棺材是早晚的事。

**Reply by devnakx** [reply]
> > [net4people/bbs#468](https://github.com/net4people/bbs/issues/468) [net4people/bbs#458](https://github.com/net4people/bbs/issues/458)
> 
> 没办法，上面都说的很直白了。HY2只适合作为备用方案用于象Racknerd或者新加坡甲骨文这种烂线路临时救急下。当然了，他可能是炫富哈~很多人还是买不起CN2 GIA的。
> 
> 无所谓了，QUIC类协议，进棺材是早晚的事。

幽默，我心想 BWG 也经常推出一些特价优惠的机子吧，我买的两个年付的 BWG VPS 总共才花七十多刀而已，咋在你嘴里就成炫富了？
![Snipaste_2025-04-11_19-48-51](https://github.com/user-attachments/assets/63c7ca57-2a9a-4379-bcb7-768dba77b9c2)

**Reply by devnakx** [reply]
> > > > > > GIA没必要用HY2吧。。
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > 为什么没必要？
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > 因为价格是你10分之一的烂线路，如果用HY2，最后速度和质量和你的GIA是一样的。反而，你好的GIA线路，因为用了HY2，还会遇到各种UDP QOS相关的症状。咋说呢。。。
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > 属于没事找事。。。
> > > 
> > > 
> > > 我这两个 vps 节点在之前一直运行 HY2 稳稳的，阻断是最近大概半年内的事，后来因为老是被间歇性阻断我就在 vps 上陆续尝试部署了trojan/vmess/vless（现在正在运行），速度都跑不满，多线程塞满了也只能跑到 50Mbps 左右，不如 HY2 一根，所以这个`没必要`因何说起？
> > 
> > 
> > 电信用户，用GIA搬瓦工一定可以跑满（别说搬瓦工这种牛逼的线路了，我用普通的9929来对付都可以跑满。跑不满，就是你的方案和设置有问题。
> 
> 典

我只是懒得和你这二极管一样的发言理论

**Reply by toyo2333** [reply]
> > > > > > > GIA没必要用HY2吧。。
> 
> > > > 
> 
> > > > 
> 
> > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > 为什么没必要？
> 
> > > > 
> 
> > > > 
> 
> > > > > 
> 
> > > > 
> 
> > > > 
> 
> > > > > 因为价格是你10分之一的烂线路，如果用HY2，最后速度和质量和你的GIA是一样的。反而，你好的GIA线路，因为用了HY2，还会遇到各种UDP QOS相关的症状。咋说呢。。。
> 
> > > > 
> 
> > > > 
> 
> > > > > 
> 
> > > > 
> 
> > > > 
> 
> > > > > 属于没事找事。。。
> 
> > > > 
> 
> > > > 
> 
> > > > 我这两个 vps 节点在之前一直运行 HY2 稳稳的，阻断是最近大概半年内的事，后来因为老是被间歇性阻断我就在 vps 上陆续尝试部署了trojan/vmess/vless（现在正在运行），速度都跑不满，多线程塞满了也只能跑到 50Mbps 左右，不如 HY2 一根，所以这个`没必要`因何说起？
> 
> > > 
> 
> > > 
> 
> > > 电信用户，用GIA搬瓦工一定可以跑满（别说搬瓦工这种牛逼的线路了，我用普通的9929来对付都可以跑满。跑不满，就是你的方案和设置有问题。
> 
> > 
> 
> > 典
> 
> 
> 
> 我只是懒得和你这二极管一样的发言理论

嘿，电信用GIA只能搞出50M，然后去和UDP的qos做斗争， 你还是去买机场或商业vpn吧，那个适合你。

**Reply by devnakx** [reply]
> > > > > > > > GIA没必要用HY2吧。。
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > > > 为什么没必要？
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > > 因为价格是你10分之一的烂线路，如果用HY2，最后速度和质量和你的GIA是一样的。反而，你好的GIA线路，因为用了HY2，还会遇到各种UDP QOS相关的症状。咋说呢。。。
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > > 属于没事找事。。。
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > 我这两个 vps 节点在之前一直运行 HY2 稳稳的，阻断是最近大概半年内的事，后来因为老是被间歇性阻断我就在 vps 上陆续尝试部署了trojan/vmess/vless（现在正在运行），速度都跑不满，多线程塞满了也只能跑到 50Mbps 左右，不如 HY2 一根，所以这个`没必要`因何说起？
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > 电信用户，用GIA搬瓦工一定可以跑满（别说搬瓦工这种牛逼的线路了，我用普通的9929来对付都可以跑满。跑不满，就是你的方案和设置有问题。
> > 
> > 
> > > 
> > 
> > 
> > > 典
> > 
> > 
> > 我只是懒得和你这二极管一样的发言理论
> 
> 嘿，电信用GIA只能搞出50M，然后去和UDP的qos做斗争， 你还是去买机场或商业vpn吧，那个适合你。

你有没有想过，我是用`trojan/vmess/vless`代理协议的时候发现速度不对，然后再用`iperf3`进行佐证的？
你以为我会和你一样没有充分测试张口就来下定论？你真的显得滑稽又搞笑
从上到下，依次为`BWG DC1机房 TCP 单线程`、`BWG DC1机房 TCP 16线程`、`BWG DC9机房 TCP 单线程`、`BWG DC9机房 TCP 16线程`
![Snipaste_2025-04-12_11-26-57](https://github.com/user-attachments/assets/48ee3554-f619-46df-8482-c7d33dcc43da)
![Snipaste_2025-04-12_11-31-04](https://github.com/user-attachments/assets/b4155eff-8587-4db4-b8d1-cef1e7876119)
![Snipaste_2025-04-12_11-27-59](https://github.com/user-attachments/assets/6ad2500d-2e1b-4eaa-9d57-70abd6de7853)
![Snipaste_2025-04-12_11-28-30](https://github.com/user-attachments/assets/815062a8-f360-4fb3-86ef-2eb323aa5565)
懒得再和你这样的人讨论任何问题，你的任何回复我不会再理，浪费我时间😅😅

**Reply by roymecat** [reply]
草，速度比我直连线路还低🤣
是不是上游拔线了？没用过BWG，之前买过大妈的LAX PRO，电信测iperf3单线程有个200Mbps左右
我现在用的直连线路iperf3单线程也有30Mbps左右

**Reply by toyo2333** [reply]
> 草，速度比我直连线路还低🤣
> 是不是上游拔线了？没用过BWG，之前买过大妈的LAX PRO，电信测iperf3单线程有个200Mbps左右
> 我现在用的直连线路iperf3单线程也有30Mbps左右

所以我说，他这种买个racknerd得了。

**Reply by devnakx** [reply]
> 草，速度比我直连线路还低🤣 是不是上游拔线了？没用过BWG，之前买过大妈的LAX PRO，电信测iperf3单线程有个200Mbps左右 我现在用的直连线路iperf3单线程也有30Mbps左右

不应该，`speedtest`测 LA 本地的服务器速度都正常（甚至超出套餐标称`1.5 Gbps`许多）
一测国内的服务器立马就拉了，甚至包括测东亚、东南亚也一样
怀疑是线路问题，但我用`Hysteria2`又确确实实能跑满，而且路由追踪看确实都是走的 CN2

LA 本地：
![Snipaste_2025-04-12_11-36-37](https://github.com/user-attachments/assets/ddf76c20-b00f-4f00-9f38-2425b2066de2)
江苏电信：
![Snipaste_2025-04-12_11-36-45](https://github.com/user-attachments/assets/7a6d881c-393b-4d0d-9297-be1c77889ed5)
香港：
![Snipaste_2025-04-12_11-40-07](https://github.com/user-attachments/assets/61076d17-0c6e-4e0e-bc66-333b407d6dd9)
东京：
![Snipaste_2025-04-12_11-41-12](https://github.com/user-attachments/assets/27920a16-ebbd-431c-a2b3-bae7f91b3da7)
新加坡：
![Snipaste_2025-04-12_11-44-51](https://github.com/user-attachments/assets/e62ade68-992f-429f-8dd7-dedb76ac3182)

路由追踪：（去程）
![Snipaste_2025-04-12_11-51-04](https://github.com/user-attachments/assets/a72a60e0-de70-4dc9-a079-8d267c26daee)
路由追踪：（回程）
![Snipaste_2025-04-12_11-52-03](https://github.com/user-attachments/assets/a832c220-ed1b-4485-994d-0cfb825de160)

**Reply by roymecat** [reply]
不是，你这国际互联也很差劲哇
合理怀疑是上游线路出问题了
抛开CN不谈，到HK和SG不应该这点速度哇

**Reply by devnakx** [reply]
> 不是，你这国际互联也很差劲哇 合理怀疑是上游线路出问题了 抛开CN不谈，到HK和SG不应该这点速度哇

就 TCP 的会这样，UDP 就能跑满，但近半年来就开始无规律阻断
目前就用过 BWG 的，而且我买的两个实例都这样，我还以为跑 TCP 的要过个中转啥的才能满速
改天我提工单去问问

**Reply by yjd** [reply]
目前使用dc1。发现企业ip策略不一样，家用走quic没问题。企业ip只要用quic协议跑一段时间必封hy2，tuic。间歇性封。过段时间好了。再连又断。断以后。用nc测试发udp包正常。也就是针对quic协议有sni封。

**Reply by devnakx** [reply]
各位，我终于找到原因了，是我 VPS 装的那个逼系统的问题，Ubuntu 22.04
我今天又心血来潮排查这个速度问题，折腾了半天、各种测试就是没啥效果
编译、安装了一堆测试工具，给我那 20G 的硬盘都塞满了
因为我从来没想过可能是系统的问题，就心想反正硬盘塞满了干脆换个系统看看吧
先试了 CentOS7，优化完内核参数、弄证书、各种代理工具安装好
接着就按例再测了下速度
好家伙，一下速度就上来了
当时我就卧槽不会真是系统的问题吧
然后就挨个换了 Debian 12、CentOS 7/9，也换了 Ubuntu 24.04
速度全都正常
而我一旦再换回 Ubuntu 22.04
速度就又变回那沟槽的 50Mbps 左右了
最后现在在用 CentOS 9

![dc1-speedtest](https://github.com/user-attachments/assets/7dc13d98-eab1-4c65-a656-88ef594649c8)
![dc1](https://github.com/user-attachments/assets/a691074f-2e82-4b3f-84e7-ac2bc32d800a)
![dc9-speedtest](https://github.com/user-attachments/assets/1aedd080-cf73-4924-bab2-24b66abc8a81)
![dc9](https://github.com/user-attachments/assets/37a1e438-04b3-45bd-9bf2-c098586e13ed)

**Reply by rebelliouswhiz** [reply]
居然还有这种原因... 不过仔细想想也十分合理...

**Reply by stqfdyr** [reply]
这么奇怪吗？

**Reply by rebelliouswhiz** [reply]
@yjd 昨天和我们 IT 聊了一下，企业和家用的 IP 出口不一样，我们这儿的外网出口都是从北京走的。我也偶尔会遇到这样的问题，但家里就没问题。

**Reply by yjd** [reply]
> @yjd 昨天和我们 IT 聊了一下，企业和家用的 IP 出口不一样，我们这儿的外网出口都是从北京走的。我也偶尔会遇到这样的问题，但家里就没问题。

最近感觉策略又加强了。家用宽带跑hy2和tuic也都封。只有tcp的存活

**Reply by stqfdyr** [reply]
@yjd 你用 ipv6 的 hy2 就行了，现阶段 ipv6 很爽

**Reply by roymecat** [reply]
我用的电信
v6和v4一样，都封
一开始能连上hy2服务端，大约1分钟就会被强制掐断，且掐断后客户端向服务端发送的所有常规UDP数据包(iperf3测试)均无法接收，服务端发送的数据包可正常接收

**Reply by devnakx** [reply]
> 我用的电信 v6和v4一样，都封 一开始能连上hy2服务端，大约1分钟就会被强制掐断，且掐断后客户端向服务端发送的所有常规UDP数据包(iperf3测试)均无法接收，服务端发送的数据包可正常接收

和我这一样，不过我这边最近没有阻断了，已经稳定运行一两周左右了

**Reply by Heworld22** [reply]
被封了，折腾了一阵又回到Vless了！

**Reply by gatspy** [reply]
用不了完蛋了。

**Reply by Hcc422** [reply]
试试换其他运营商网络，我用移动不行，换电信就能用

**Reply by plok6325** [reply]
最近封的很厉害，上海电信和深圳广电宽带 都封了。 手机移动网络是正常的

**Reply by lovesans** [reply]
辽宁也封的紧,三网全一个样,用用就没了,过个10分钟左右又会放出来,反复碾压,无奈又换回VLESS了

**Reply by Hcc422** [reply]
我好起来了，都可以用

**Reply by darkaaa** [reply]
惨啊不能用了。就这几天的事。

**Reply by hongkevin233** [reply]
我觉得是udp协议开始被严厉打击了

**Comment by duzefu** [comment]
坐标深圳,服务器cloudcone ,最近一周开始无法使用, Iperf测试发现发往外网的UDP包会全部被丢弃100%丢包. 改用hysteria1 faketcp目前还可以用

**Reply by skill7899** [reply]
坐标广州，buyVm cn线，联通最近一周开始无法使用，改用hysteria1 faketcp目前还可以用。

**Reply by Sakamakiiizayoi** [reply]
坐标浙江，电信-日本东京iij 最近开始ipv4 hysteria2无法使用 。换什么端口都不行，十次有九次超时。但是ipv6 hysteria2依旧稳如老狗。

**Reply by hongkevin233** [reply]
坐标深圳，服务器hk，移动正常使用，挂了自己的域名ipv4 hy2目前还是可以使用，没有端口跳跃

**Reply by hongkevin233** [reply]
目前应该是加大了对udp协议打击

**Comment by yjd** [comment]
hy有没有可能用JLS替换。最近用shadowquic，sni拦截问题解决

**Reply by yjd** [reply]
https://github.com/apernet/hysteria/issues/1301#issuecomment-2987223298

**Comment by asharca** [comment]
带宽上传200m，下载1000m，使用hy2后上传是没问题的，下载速度却只有200m了，服务端没做带宽限制，客户端的带宽限制如下。

```yaml
bandwidth:
  up: 200 mbps
  down: 1000 mbps
```

**Reply by asharca** [reply]
> 带宽上传200m，下载1000m，使用hy2后上传是没问题的，下载速度却只有200m了，服务端没做带宽限制，客户端的带宽限制如下。
> 
> ```yaml
> bandwidth:
>   up: 200 mbps
>   down: 1000 mbps
> ```

测试下来发现路由器上显示的流量跑满1000mbps，但是走hysteria2机器上的流量只有100-200mbps，这是为啥

**Comment by Levitans** [comment]
我运行hy2的vps在Let's Encrypt申请的免费证书，客户端我用的sing-box，我在mac上能正常使用，但是在windows上需要设置tls的insecure为ture否则没法使用。
我知道这不是hy2的问题，但我找不到解决办法，有没有大佬解释一下为啥🥹

**Reply by Sakamakiiizayoi** [reply]
使用完整证书链的证书试试

**Reply by Levitans** [reply]
可以了，完美解决，非常感谢。我第一次给服务器配置证书没注意这个问题

**Reply by iddsjo** [reply]
> 使用完整证书链的证书试试

感谢，解决了

**Comment by rebelliouswhiz** [comment]
我最近在调试网站，443 端口上的 QUIC。我发现网站怎么着都会有点问题，薛定谔猫般状态。我在 v2ex 上发了个贴：https://www.v2ex.com/t/1171036

但我很奇怪，同样基于 QUIC 的 hysteria2 为什么不受影响？

首先我确保了域名没被墙，确保了服务器没被墙，确保了网站真的支持社会主义真善美，但 443 端口 QUIC 访问网页就是有问题，而同区域另一台服务器在 443 的 hysteria2 就没问题。

我有如下假设：

- 如果你同时在 443 监听 tcp 和 udp，你就会被 gfw 照顾；
- 如果你不监听 tcp，域名和服务器没问题的情况你就会被放过；
- GFW 没时间监听那么多服务器/域名的 udp 端口，一台服务器 65535 个端口，那得多少资源扫描和监控。

基于如上假设，再基于 https://gfw.report/publications/usenixsecurity25/zh/#3.3 的发现，如果你监听 65535 端口，应该要比监听 443 端口要来的安全。

以上假设未完全验证，我 hysteria2 长期跑在 30000-40000 端口好像也没啥问题... 但是国内访问网站 QUIC 总是被干扰，搞得我已经放弃 QUIC，全部 http2 极为稳定...

**Reply by toyo2333** [reply]
就这么说吧。UDP就是投机取巧。特殊情况下拿来做backup可以。日常就算了。

**Reply by blackboxs777** [reply]
udp快啊

**Reply by DelimChan** [reply]
现在封得很严重，我都准备换协议了，快不如稳定。

**Reply by dcdebug** [reply]
有的服务器正常，有的有异常，不知道是不是设置有问题，以前是没有问题的。

**Comment by Bemly** [comment]
<img width="2216" height="2906" alt="ee6f64f4d2c5ff116d704b0cd0281749" src="https://github.com/user-attachments/assets/c3757b04-7538-496c-a824-4b556fd98ee1" />
<img width="1320" height="506" alt="QQ_1775896683064" src="https://github.com/user-attachments/assets/f90dfde8-5cfb-42f0-8e55-b028b01cdbea" />
<img width="1358" height="258" alt="QQ_1775896710706" src="https://github.com/user-attachments/assets/df785816-5983-45ea-ab10-46a17887d3de" />

ports: 50000-63491 port: 63492 sni: www.bing.com skip-cert-verify: true
用了2年多没问题，平时偶尔看看片又快又稳，昨天心血来潮去AMD下个Xilinx开发套件，平均100Mbps快40G的时候把哥们ipv4入方向全部封了，现在只剩v6了，上传只有1Mbps😭

**Reply by iddsjo** [reply]
为什么都喜欢偷bing

**Reply by Bemly** [reply]
> 为什么都喜欢偷bing

一般用啥嘞，是bing的缘故嘛

**Reply by Bemly** [reply]
隔了3天ip又解回来了，很奇怪

**Reply by Mcdouglaslock** [reply]
难道流量大了会增大被封的风险吗？你用的是自签名证书，还是用的域名申请的let's encrypt

**Reply by Bemly** [reply]
> 难道流量大了会增大被封的风险吗？你用的是自签名证书，还是用的域名申请的let's encrypt

bzd，换reality偷了，感觉是流量大了被限了，可能sni也有点关系，我用的www.bing.com被封的，可能自签名证书、自申请没这个问题

**Reply by Mcdouglaslock** [reply]
我也换reality偷了，但是我看有人说偷大厂的域名好像也不是特别安全，容易物以类聚，可以选择reality偷自己，比如这篇博客里说的
https://blog.acesheep.com/p/xray-reality-protocol/  ，但是得有个域名才行。
我目前就采用的这个方案，还可以。另外hy2协议我也是用域名申请的let's encrypt,然后开了端口跳跃，目前来讲常规用没出过问题，但也不敢一下子搞几十个G的流量🫠
